shell/common/animator.h

```diff
@@ -52,6 +52,8 @@ class Animator final {
 
   void Render(std::unique_ptr<flutter::LayerTree> layer_tree);
 
+  void ScheduleSecondaryVsyncCallback(std::function<void()> callback);
```

Comment:
Add docstring.

Reply:
I read the docstring later. Can you also repeat the same thing here for clarity without having to jump through links.